.github/workflows/ci.yml

```
@@ -0,0 +1,41 @@
+name: CI
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+  schedule: [cron: "40 1 * * *"]
+
+permissions:
+  contents: read
+
+jobs:
+  install:
+    name: Noir ${{matrix.version}} on ${{matrix.os == 'ubuntu' && 'Linux' || matrix.os == 'macos' && 'macOS' || matrix.os == 'windows' && 'Windows' || '???'}}
+    runs-on: ${{matrix.os}}-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu, macos]
+        version: [0.1.0, 0.1.1]
```

It'd be cool if we could test without a version

---

Hmm, yes. The `include` array should be doing this but it doesn't seem like it is.

This can be handled by having an explicit default value for the `version` key so as that's already planned then I don't think we need to worry about this.

---

I wonder if github trims the `version` key if you use something like:
```suggestion
        version: ["", "0.1.0", "0.1.1"]
```